cdc/owner/changefeed.go

```diff
@@ -432,7 +452,7 @@ LOOP:
 	}()
 
 	stdCtx := contextutil.PutChangefeedIDInCtx(cancelCtx, c.id)
-	redoManagerOpts := &redo.ManagerOptions{EnableBgRunner: false}
+	redoManagerOpts := &redo.ManagerOptions{EnableBgRunner: true}
```

Comment:
Ref: https://github.com/pingcap/tiflow/blob/master/cdc/owner/changefeed.go#L527

Reply:
At that place, the redoManager is created for clean something after the changefeed has been removed. So `false` is OK.